Azure/communication-ui-library-android
# [Calling][Bugfix] Fix issue for controls 'Android' and 'Partecipa alla chiamata' are not visible in large text size on 'Configura' screen

## Purpose
When changing to italian in App settings and modify the text size to largest in accessibility, we want to make sure the text on setup screen are visible.

## Does this introduce a breaking change?


## Pull Request Type
What kind of change does this Pull Request introduce?


## Pull Request Checklist




## How to Test
Open Application
Click on 'Setting' button and Select 'Language' to 'it-it'.
Navigate to ACS Token Button and activate.
Fill out required details like ACS token in 'ACS Token' test field and name in 'display name' text field.
Navigate to 'Group Call' activate it and navigate to Group ID edit field.
Enter 'Group Call' id in text field of Group ID edit field
Navigate to 'Join' button.
Configura Screen will be appeared.
Observe the issue in large text size.
